src/main/java/App.java

```diff
@@ -1,18 +1,18 @@
-
 import java.io.FileNotFoundException;
 import java.io.FileReader;
 import java.io.IOException;
 import org.apache.spark.sql.*;
 import org.json.simple.JSONObject;
 import org.json.simple.parser.JSONParser;
 import org.json.simple.parser.ParseException;
+import hdfs.HDFS;
 
 /**
  * Hello world!
  *
  */
 public class App {
-
+    private static HDFS hdfs = new HDFS();
```

Comment:
Need to move implementation into a new HDFSImporter.java class that extends Connector